### PR TITLE
Armour rebalance

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
TL;DR I am ruining christmas by decreasing light armour values by 5 across the board, and increasing heavy fire, bio and bomb by 5.

This PR aims to make armour class somewhat more meaningful, and is mainly targeted at armour scaling vastly benefiting light armour.

Currently light armour has a slowdown of 0.3 (compared to 0.5 for medium, 0.7 for heavy) but its armour is only 5 lower than medium. This means going from light armour to medium increases your slowdown by 67% for a 12.5% increase in (melee) armour.

Speed is probably the most important stat in the game, both for running away and chasing down kills, so current armour scaling means there is very low opportunity cost for taking light armour. After the change, light armour still has a vastly better armour to slowdown ratio (it's like 1.5 extra damage from an ancient rav slash), but should at least be slightly less wack.

Heavy armour is a bit different because you're basically accepting you're not going to outrun anything, and higher armour has more efficient scaling, especially when you add on upgrades like tyr.

It also never made much sense that fire, bomb and bio armour didn't scale (and this also further benefits light armour), so I have increased heavy armour for these values by 5.
## Why It's Good For The Game
Makes the huge benefits of light armour beno chasing slightly more costly.
## Changelog
:cl:
balance: Light armor armor values reduced by 5 across the board
balance: Heavy armor fire, bio and bomb armor increased by 5
/:cl:

### CODE REVIEW

**Comment by Losenis** [comment]
You get looked at funny in light armor and you already get delimbed and 80 points of IB with a slowdown by pain heavier than getting hit thrice in heavy armor, what are you smoking? Light armor is only good if you don't get hit at all which can also be said about going with no armor. This only makes light armor even more suicidal and insta-DNR when you get sneezed at and suddenly you're an expired ketchup packet ran over by a 18-wheeler. I don't know how to tell you this but you can just go without armor for even more speed, but we all know that's not going to get touched at because it's just as instadeath as going with light armor when you actually get hit.

A bigger problem is that some xenos are just stupidly slow overall with little to no tools to stop chasers and get chased even by people on heavy armor, like boiler or off-weeds HL, or anything hit by a grenade.

**Comment by Barnet2** [comment]
> You get looked at funny in light armor and you already get delimbed and 80 points of IB with a slowdown by pain heavier than getting hit thrice in heavy armor, what are you smoking? Light armor is only good if you don't get hit at all which can also be said about going with no armor. This only makes light armor even more suicidal and insta-DNR when you get sneezed at and suddenly you're an expired ketchup packet ran over by a 18-wheeler. I don't know how to tell you this but you can just go without armor for even more speed, but we all know that's not going to get touched at because it's just as instadeath as going with light armor when you actually get hit.
> 
> A bigger problem is that some xenos are just stupidly slow overall with little to no tools to stop chasers and get chased even by people on heavy armor, like boiler or off-weeds HL, or anything hit by a grenade.

Completely agree with this sentiment. Playing xeno regularly most light armor users are but a joke given the tile-base nature of ss13 making it pretty easy to hit them with abilities. This PR does make medium armor less of a noob trap, but I don't think this change is warranted. A good player in light armor is as scary as in heavy armor, to me.

**Comment by FIXTHERADBOXPLEASE** [comment]
Light armor is, quite literally, not even two damage more than taking medium armor in 99% of cases. Any perceived differences in protection are placebo, or light armor's mobility allowing you to get in situations where you can be damaged. 
That's not exactly a true downside.

**Comment by Lumipharon** [comment]
> You get looked at funny in light armor and you already get delimbed and 80 points of IB with a slowdown by pain heavier than getting hit thrice in heavy armor, what are you smoking? Light armor is only good if you don't get hit at all which can also be said about going with no armor. This only makes light armor even more suicidal and insta-DNR when you get sneezed at and suddenly you're an expired ketchup packet ran over by a 18-wheeler. I don't know how to tell you this but you can just go without armor for even more speed, but we all know that's not going to get touched at because it's just as instadeath as going with light armor when you actually get hit.
> 
> A bigger problem is that some xenos are just stupidly slow overall with little to no tools to stop chasers and get chased even by people on heavy armor, like boiler or off-weeds HL, or anything hit by a grenade.

The difference in armour is like 1 damage more per hit.

**Comment by Losenis** [comment]
If you manage to get hit only once, the argument is made irrelevant by the existence of arrowhead that doesn't have slowdown. If you manage to get hit several times, get fucked because it's not just damage, you're crossing the threshold of ending up with more IB, or get unlucky enough to end up with that theoretical 1 point of damage in that one single limb to frac it rather than ending up with lower damage RNGd to another limb to prevent the frac altogether. If you get jumped by a runner you'll get hit thrice guaranteed unless you're in an ungaball, and say goodbye to your limbs if you get stunned by a rav. Or god forbid, a shrike stun-displaces you into a group of xenos.

Marines dont have xeno health system, it isn't purely the funny green healthbar that they have to worry about.

**Comment by Losenis** [comment]
You only need to see how the all-caps guy is agreeing with it to realize how dumb the idea is

**Comment by Pariah919** [comment]
I'd honestly rather see LA lose acid than lose melee, I don't think there's a massive need to nerf it in the reality of TGMC even if paper stats say otherwise. As someone who basically has used medium armor since ever.

**Comment by Lumipharon** [comment]
> If you manage to get hit only once, the argument is made irrelevant by the existence of arrowhead that doesn't have slowdown. If you manage to get hit several times, get fucked because it's not just damage, you're crossing the threshold of ending up with more IB, or get unlucky enough to end up with that theoretical 1 point of damage in that one single limb to frac it rather than ending up with lower damage RNGd to another limb to prevent the frac altogether. If you get jumped by a runner you'll get hit thrice guaranteed unless you're in an ungaball, and say goodbye to your limbs if you get stunned by a rav. Or god forbid, a shrike stun-displaces you into a group of xenos.
> 
> Marines dont have xeno health system, it isn't purely the funny green healthbar that they have to worry about.

Fractures are based on a damage threshold. That means SOME xenos will frac in 1 hit less, the rest will be unchanged. But hey, luckily you aren't forced to wear light armour. Imagine there being an actual trade off between speed and survivibility?

**Comment by Losenis** [comment]
>Fractures are based on a damage threshold.

Has anyone said otherwise? A hit vs light armor might skirt the damage threshold and frac in the hit while on medium armor you might end up just under the threshold, and the next hit might RNG into another limb. This is a massive difference let alone adding IB on top of it. If you get hit with light armor, you run into way more issues than just seeing a funny number on healthscan.

>Imagine there being an actual trade off between speed and survivibility?

If you're implying there already isn't, then I don't know what game you're playing. Probably one where chem machine is powergaming somehow?

**Comment by FIXTHERADBOXPLEASE** [comment]
> A hit vs light armor might skirt the damage threshold and frac in the hit while on medium armor you might end up just under the threshold

The armor thresholds are not balanced that way. You are experiencing the placebo effect.

**Comment by Losenis** [comment]
Basic math and the actual game beg to differ.

**Comment by FIXTHERADBOXPLEASE** [comment]
The "actual game" referring to your subjective experiences that do not match any raw calculations or in-game tests I've performed on armored marines I've spawned in.

**Comment by Losenis** [comment]
And that's coming from the guy that went around making scenarios up like suiciding after losing one of three PCs or some other nonsense like that? Comedy gold

**Comment by FIXTHERADBOXPLEASE** [comment]
Please no shit flinging in the PR...

**Comment by iforgotmeword** [comment]
Let’s just test it to get it over with

**Comment by SilencedMP5A5** [comment]
hmm yes even more reason to just run no armor at all

**Comment by maikilangiolo** [comment]
> This PR does make medium armor less of a noob trap,

It actually doesn't. It makes light armor more of a noob trap (because it's very, very risky and so easy to be killed), medium armor is untouched (and still awful), but heavy armor has become the very best option. Zero reason to use medium, zero reason to use light.

File: code/modules/clothing/modular_armor/attachments/chest_plates.dm [hunk 1]
```diff
@@ -30,7 +30,7 @@
 	name = "\improper Jaeger Pattern Light Skirmisher chestplates"
 	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides minor protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Skirmisher armor piece."
 	icon_state = "skirmisher"
-	soft_armor = list(MELEE = 20, BULLET = 40, LASER = 40, ENERGY = 30, BOMB = 30, BIO = 30, FIRE = 30, ACID = 30)
+	soft_armor = list(MELEE = 15, BULLET = 35, LASER = 35, ENERGY = 25, BOMB = 25, BIO = 25, FIRE = 25, ACID = 25)
```
Comment (carlarctg) [inline code/modules/clothing/modular_armor/attachments/chest_plates.dm:33]: Honestly you could just make these defines at this point

#define LIGHT_ARMOR_SOFT_PROTECTION list(...)

would help readability a liiiittle bit and make changes like these less annoying

**Comment by TChalkman** [comment]
You get a free frac or even a delimb with light armor by any roonie, warrior or hunter that pounces on you. 
Anything with a lot of melee damage like a ravager can give you a free frac with light armor, really.

Just ask people like Ocelot, who needs to chug oxycodone so they're not a literal ragdoll after being tackled once (1) by a roonie.

Bad PR. If you want a greater difference, buff the armor values of medium armor, instead of making the trade of speed unbearable. INB4 we get rounds of turtling heavy armor marines with m60s only.

**Comment by RaidarV** [comment]
>use light armor
>start chasing xeno
>xeno touches you
>get frac, IB, de-limbed
>die

true story !

**Comment by Joe13413** [comment]
> Just ask people like Ocelot, who needs to chug oxycodone so they're not a literal ragdoll after being tackled once (1) by a roonie.

ok to be fair savaged by a rouny + 2-3 slashes does like half a marines health and sometimes breaks bones, not really surprised that the dedicated hit and run caste can do hella damage on a hit and run attack

**Comment by Al-1ce** [comment]
> You get a free frac or even a delimb with light armor by any roonie, warrior or hunter that pounces on you. Anything with a lot of melee damage like a ravager can give you a free frac with light armor, really.

You exchange speed and mobility for lower protection. That's a given. If you wear light armor, expect to get hurt easily. That's something _you_ should account for, not the game. The idea itself works as intended.

> Just ask people like Ocelot, who needs to chug oxycodone so they're not a literal ragdoll after being tackled once (1) by a roonie.


What you're forgetting here is playstyle. Ocelot likes to run into the middle of hive wearing next-to no armor, and often ends up getting swarmed. 

Even if they did this with heavy armor, they'd still be putting themself into a very bad position. The game discourages this kind of behavior; the few people who can pull it off successfully are notorious and revered as a result. 


> Bad PR. If you want a greater difference, buff the armor values of medium armor, instead of making the trade of speed unbearable. INB4 we get rounds of turtling heavy armor marines with m60s only.


I actually agree with this. Light armor is already paper-thin.

**Comment by Jordanstolethepreciousslam** [comment]
I don't think heavy armor needs buffs. It's underestimated by a lot of people. Light armor is good for individual performance, but heavy armor wins games with a solid frontline and team effort. Nerfing melee values also a double edge decision, sure, it would make an effect by to be entirely honest , IBs and fracs are more of a nuisance rather than a valid reason to reconsider your choice of armor. But for newer players they are death sentence. What is really should be treated, is acid protection powercreep. Larmor have no business casually having 50% reduction, given that burn damage is nowhere painful it was, leaves no lasting damage and is healed separately from brute damage, meaning you heal hybrid damage twice as fast you heal single type damage.

**Comment by Mannybrado** [comment]
woah, a marine nerf? no no no, we need BUFFS for marines. clearly!

**Comment by maikilangiolo** [comment]
Someone mentioned making light armor bio and acid lower (keeping melee as is) and buffing medium acid and bio to be better than heavy. 
I feel like that would be a better compromise. Light is sufficiently weak against melee as-is, but could be countered by acid and neuro better. Medium would get a niche of improve acid resistance since it's protects for 1 extra slash vs light.

But hey, not like player suggestions matter, am I right?

**Comment by Liznerd** [comment]
Man light armor is not that great, the pain you get from the lack of protection ends up slowing you like a heavy armor guy, broken bones, organ damage are things common too, i agree that the gameplay relies heavily on speed rather than tactics or stats, that's what makes the armor the most popular one, not because it is broken, but because speed is the most important thing in the game

**Comment by RaidarV** [comment]
how will nerfing light armor and buffing heavy armor make medium armor better/more useful ?
